api/tests/integration/edge_api/identities/test_edge_identity_featurestates_view.py

```diff
@@ -253,6 +290,71 @@ def test_edge_identities_create_featurestate(
     assert args[0] == expected_identity_document
 
 
+def test_edge_identities_with_identifier_create_featurestate(
+    admin_client,
+    environment,
+    environment_api_key,
+    identity_document_without_fs,
+    dynamo_wrapper_mock,
+    feature,
+):
+    # Given
+    dynamo_wrapper_mock.get_item.return_value = identity_document_without_fs
+    identity_identifier = identity_document_without_fs["identifier"]
+    url = reverse(
+        "api-v1:environments:edge-identities-with-identifier-featurestates-list",
+        args=[environment_api_key],
+    )
+    expected_feature_state_value = "random_value"
+    expected_fs_enabled = True
+    data = {
+        "multivariate_feature_state_values": [],
+        "enabled": expected_fs_enabled,
+        "feature": feature,
+        "feature_state_value": expected_feature_state_value,
+        "identifier": identity_identifier,
+    }
+
+    # When
+    response = admin_client.post(
+        url, data=json.dumps(data), content_type="application/json"
+    )
+
+    # Then
+    assert response.status_code == status.HTTP_201_CREATED
+    assert response.json()["feature"] == feature
+    assert response.json()["feature_state_value"] == expected_feature_state_value
+
+    dynamo_wrapper_mock.get_item.assert_called_with(
+        f"{environment_api_key}_{identity_identifier}"
+    )
+    name, args, _ = dynamo_wrapper_mock.mock_calls[1]
+    assert name == "put_item"
+
+    # Next, let's verify that the document that we put
+    # have the fs that we created
+
+    # First, let's create the copy of the original document
+    expected_identity_document = identity_document_without_fs
```

Comment:
This is a bit misleading since it isn't actually copying anything, it's just creating a pointer to the original variable (if I understand python correctly).